logutils/override.go

```diff
@@ -10,7 +10,8 @@ import (
 )
 
 type LogSettings struct {
-	Enabled         bool   `json:"Enabled"`
+	Enabled bool `json:"Enabled"`
+	// Deprecated: MobileSystem should not be used and will be removed in the future.
 	MobileSystem    bool   `json:"MobileSystem"`
```

Comment:
It will break compatibility with mobile, which currently uses `InitiLogging endpoint`
So it's better to deprecate now and safely remove later.

Reply:
@qfrank Well, because you're passing `{ "MobileSystem": false }`, which will not be able to unmarshal into `LogSettings`, if we remove this field 🤔 

https://github.com/status-im/status-mobile/blob/87483b1ef494176c07d18933cb8534237dd8d55f/src/status_im/common/log.cljs#L29

Reply:
hmmm, looks like it's not an error indeed. Will remove it then  👍 
https://go.dev/play/p/X_-SBKNs7JO

Reply:
Removed: 0bfae2fb9eed859842c5d31331c974602dd5bea7